specs/howto.md

```diff
@@ -0,0 +1,358 @@
+* `description`: A textual description of the entity.
+* `dataProvider`: A name identifying the entity providing the data.
+
+Similarly most of the Data Models extends the Location common schema, and thus
+include the following attributes:
+* `address`: the civic address of the entity as per 
+  [schema.org](http://schema.org/address).
+* `location`: the [GeoJSON](https://tools.ietf.org/html/rfc7946)
+  representation of the entity location.
+
+## General tips
+
+### How to specify unit codes
+
+1. If your data use the default unit defined in the Data Model, you don't
+  need to specify any. It is implied.
+1. If your data use a different unit, you will need to use the `unitCode`
+  metadata annotation in your data (and you will need to adopt the normalised
+  representation). Code used should be the ones defined by
+  [UN/CEFACT](https://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex3e.pdf). 
+  E.g.:
+
+  ```json
+  "length": {
+      "value": 11,
+      "metadata": {
+          "unitCode": {
+              "value": "FOT"
+          }
+      }
+  }
+  ```
+    
+
+### How to attach specific time of update to a single attribute
+
+Often, different attributes of the same entity are updated at different points
+in time as results of the aggregation of data generated from different sensors.
+To be able to define when a specific attribute has been updated (that anyhow
+cannot correspond to the last `dateModified` value that reflect only the
+point in time the NGSI v2 implementation received an entity update),
+you should use the `timestamp` metadata annotation. E.g.:
+
+```json
+"speed": {
+    "value": 11,
+    "metadata": {
+        "timestamp": {
+            "value": "2016-03-15T11:00:00.000"
+        }
+    }
+}
+```
+
+### Proper usage of relations among Data Models
+
+In several models, and more and more with the introduction of NGSI-LD, you will
+find attributes that actually represent relationships with other entities.
+
+Previously the FIWARE data model defined the name of such attributes was normed
+to be `ref` followed by the name of the Entity Type referenced by the attribute,
+for example in the case of Entity Type `Device`, the full attribute would
+be `refDevice`.
+
+With the introduction of NGSI-LD, following the link with practises in the
+ontologies, such attributes are defined with the usage of a verb (plus
+optionally an object) such as `hasStop`, `operatedBy`, `hasTrip`, etc.
+
+While in the first case, the entity type to be looked for is implicitly defined
+by the attribute name, this is no longer the case with NGSI-LD, that's why it
+is important to use NGSI-LD [URNs](https://tools.ietf.org/html/rfc8141)
+that convey the type of the target entity,
+for instance `urn:ngsi-ld:gtfs:Stop:S123`.
+
+Currently it is up to the logic of your application to include the logic to
+navigate entity relationships, as it is not yet defined within implementations
+of the NGSI v2 or NGSI-LD specification.
+
+## TODO
```

Comment:
Still waiting for the **TODO**

Reply:
As mentioned above, for the TODO, to avoid people to have to fork the whole repo to test their examples, I need first to introduce support for remote schema in the validator. It will take time. Better to close this PR and open a new one when remote schema support is introduced.

Reply:
In that case LGTM